frontend/src/components/common/select/select.tsx

```diff
@@ -1,7 +1,15 @@
 import SelectReact from 'react-select';
 import { Option } from 'common/types/types';
 import { ErrorMessage } from '@hookform/error-message';
-import { Control, FieldErrors, Path, FieldValues } from 'react-hook-form';
+import {
+  Control,
+  FieldErrors,
+  Path,
+  FieldValues,
+  useController,
+} from 'react-hook-form';
+import { useState } from 'hooks/hooks';
+import { FIRST_ARRAY_IDX } from 'jabber-shared/common/constants/constants';
```

Comment:
It will be better if you add export 'FIRST_ARRAY_IDX' in 'frontend/src/common/contants' and then import from it